pkg/cache/server/crd_lister.go

```diff
@@ -18,19 +18,41 @@ package server
 
 import (
 	"context"
+	"fmt"
+
+	"github.com/kcp-dev/logicalcluster/v2"
 
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	apiextensionslisters "k8s.io/apiextensions-apiserver/pkg/client/listers/apiextensions/v1"
 	"k8s.io/apiextensions-apiserver/pkg/kcp"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
 
 	"github.com/kcp-dev/kcp/pkg/cache/server/bootstrap"
 	"github.com/kcp-dev/kcp/pkg/client"
 )
 
+// crdClusterLister is a CRD lister
+type crdClusterLister struct {
+	lister apiextensionslisters.CustomResourceDefinitionLister
+}
+
+func (c *crdClusterLister) Cluster(name logicalcluster.Name) kcp.ClusterAwareCRDLister {
+	if name != bootstrap.SystemCRDLogicalCluster {
+		klog.Background().Error(fmt.Errorf("cluster-unaware crd lister got asked for %v cluster", name), "programmer error")
```

Comment:
why do we need this?

Reply:
Hmmm ... good question - it must have come from somewhere previous. Do you know? :D

Reply:
Looks like it's only ever valid to use this particular cluster - maybe we don't need a `Cluster()` call here at all, let me remove

Reply:
(and it was your first-pass impl that hard-coded this value, and has a TODO for future improvements ;) )

Reply:
i'd like to see how the apiextention server will be using `crdClusterLister`, could you please provide me with a link to your fork?

Reply:
okay, clarified with Steve in a gmeet.